<launch>

  <arg name="uav_name" default="uav" />


    <arg name="gui" default="true" />

    <node name="balloon_color_picker" pkg="balloon_color_picker" type="color.py" output="screen">


      <!-- Subscribed topics -->
      <remap from="image_topic" to="/uav/rs_d435/color/image_raw" />
      <!-- Published topics -->
      <remap from="circle_topic" to="~circled" />
      <remap from="circle_hsv" to="~circled_hsv" />
      <remap from="circle_lab" to="~circled_lab" />
      <!-- Params -->
      <param name='save_dir' value='$(find object_detect)/config/'/>
      <param name='config_path' value='$(find object_detect)/config/balloon_config.yaml'/>


    </node>
    <!-- Gui Node -->
    <node name="balloon_color_picker_gui" pkg="balloon_color_picker" type="plugin.py" output="screen">
    </node>
    

</launch>
